packages/nextjs/package.json

```diff
@@ -11,7 +11,8 @@
     "format": "prettier --write . '!(node_module|.next|contracts)/**/*'",
     "check-types": "tsc --noEmit --incremental",
     "vercel": "vercel",
-    "vercel:yolo": "vercel --build-env NEXT_PUBLIC_IGNORE_BUILD_ERROR=true"
+    "vercel:yolo": "vercel --build-env NEXT_PUBLIC_IGNORE_BUILD_ERROR=true",
+    "spheron": "next build && next export && spheron login --github && spheron publish"
```

Comment:
Can we instead use `spheron upload --path ./out --protocol ipfs` and omit the `spheron.json` file? I prefer to keep the optional config files to a minimum to avoid potentially confusing users.